spec/cloudtasker/backend/redis_task_spec.rb

```diff
@@ -242,36 +243,74 @@
     subject { task.deliver }
 
     let(:status) { 200 }
-    let!(:http_stub) do
-      stub_request(:post, job_payload.dig(:http_request, :url))
-        .with(
-          headers: {
-            Cloudtasker::Config::TASK_ID_HEADER => task_id,
-            Cloudtasker::Config::RETRY_HEADER => job_payload[:retries]
-          },
-          body: job_payload.dig(:http_request, :body)
-        )
-        .to_return(status: status)
-    end
 
-    before do
-      allow(task).to receive(:destroy).and_return(true)
-      allow(task).to receive(:retry_later).with(described_class::RETRY_INTERVAL).and_return(true)
+    context 'with HTTP' do
```

Comment:
You can probably meta-generate the specs to avoid repetition 😉 

E.g.
```ruby
['http', 'https'].each do |tested_protocol|
  context "with #{tested_protocol}" do
    let(:url) { "#{tested_protocol}://localhost:300/run" }
    ...
  end
end
```

Reply:
Fixed.